app/models/katello/content_view_version.rb

```diff
@@ -109,6 +109,14 @@ def description
       history.publish.successful.first.try(:notes)
     end
 
+    def description=(new_description)
+      if description
```

Comment:
It wouldn't necessarily mean that the publish had failed.  That said, I'm okay with an exception being thrown for this condition.  

your comment did make me realize that your code won't work though, as the history event isn't marked as successful until after you've tried to save the description:

https://github.com/akofink/katello/blob/608c06fccc7ee3ee78e1527bce8e6f8cfaebc5fb/app/lib/actions/katello/content_view_version/incremental_update.rb#L143-L147

why not just pass the history item into that method and update it there rather than worrying about a 'description=' method?